docs/en/oas3/OAS3-PDND-AS.yaml

```
@@ -0,0 +1,313 @@
+        - name: Digest
+          in: header
+          description: >-
+            Digest of the message payload, to comply with the INTEGRITY_REST_02
+            security pattern. According to <a target="blank" href="https://www.rfc-editor.org/rfc/rfc3230.html#section-4.2">RFC
+            3230 §4.2</a>, the format MUST be the following: digest-algorithm=encoded
+            digest output.
+          required: true
+          schema:
+            type: string
+            example: SHA-256=b626aa33e25d46086dd5c12b631be51caeece92689912408e96dc4a430f3fa67
+        - name: Agid-JWT-TrackingEvidence
+          in: header
+          description: >-
+            JWT containing the data tracker in the Consumer's domain. It is mandatory only when complying with AUDIT_REST_02.
+            <a target="blank"
+            href="https://italia.github.io/eudi-wallet-it-docs/versione-corrente/en/e-service-pdnd.html">e-Service PDND</a>. <br/><br/>
+
+            <a target="blank" href="https://jwt.io/#debugger-io?token=eyJhbGciOiJFUzI1NiIsImtpZCI6IjhlOGRlNGJmZTFlNWVhNzkwM2E1ODE2ZmNlNzNjZDQ4IiwidHlwIjoiSldUIn0.eyJpc3MiOiJpbnRlcm9wLnBhZ29wYS5pdCIsImF1ZCI6Imh0dHBzOi8vZXJvZ2F0b3JlLmV4YW1wbGUvZW50ZS1leGFtcGxlL3YxIiwiZXhwIjoxNzgzMDQyMTUwLCJuYmYiOjE3MzMwNDE5NDUsImlhdCI6MTczMzA0MTkyMCwianRpIjoiYzRmNWQ3ZTItYjdjOC00MGY2LTliNmEtZGM5YTRmNWFlYjU3IiwiY2xpZW50X2lkIjoiODI5MTRiM2YtNjBiMi00NTI5LWI0ZDYtM2Q0ZTY3ZjBhOTMzIiwicHVycG9zZUlkIjoiZDJiOWE2NTMtYzQ5Ny00NWM2LWI4ZjEtNWJkZjEyNGM5ZDNhIiwiZG5vbmNlIjoiNjUyODQyNDIxMzY4NSIsInVzZXJJRCI6ImE4YjdjNmQ1LWU0ZjMtZzJoMS1pOWowLWtsbW5vcHFyc3R1diJ9.vHispWBZeWk0UYwUSMAxpioC4giifbctyC3sgiqNxNHt_ypxIr6FZkKlTrLc02WivSm3ApbZ0ABi1RgtcUthYQ">EXAMPLE
+            ON JWT.IO</a>
+          required: true
+          schema:
+            type: string
+            format: JWT
+            example: eyJhbGciOiJFUzI1NiIsImtpZCI6IjhlOGRlNGJmZTFlNWVhNzkwM2E1ODE2ZmNlNzNjZDQ4IiwidHlwIjoiSldUIn0.eyJpc3MiOiJpbnRlcm9wLnBhZ29wYS5pdCIsImF1ZCI6Imh0dHBzOi8vZXJvZ2F0b3JlLmV4YW1wbGUvZW50ZS1leGFtcGxlL3YxIiwiZXhwIjoxNzgzMDQyMTUwLCJuYmYiOjE3MzMwNDE5NDUsImlhdCI6MTczMzA0MTkyMCwianRpIjoiYzRmNWQ3ZTItYjdjOC00MGY2LTliNmEtZGM5YTRmNWFlYjU3IiwiY2xpZW50X2lkIjoiODI5MTRiM2YtNjBiMi00NTI5LWI0ZDYtM2Q0ZTY3ZjBhOTMzIiwicHVycG9zZUlkIjoiZDJiOWE2NTMtYzQ5Ny00NWM2LWI4ZjEtNWJkZjEyNGM5ZDNhIiwiZG5vbmNlIjoiNjUyODQyNDIxMzY4NSIsInVzZXJJRCI6ImE4YjdjNmQ1LWU0ZjMtZzJoMS1pOWowLWtsbW5vcHFyc3R1diJ9.vHispWBZeWk0UYwUSMAxpioC4giifbctyC3sgiqNxNHt_ypxIr6FZkKlTrLc02WivSm3ApbZ0ABi1RgtcUthYQ
```

See previous comment.

```suggestion
            example: eyJhbGciOiJFUzI1NiIsImtpZCI6ImQ0YzNiMmExLTk4NzYtNTQzMi0xMGZlLWRjYmE5ODc2NTQzMiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiI4MjkxNGIzZi02MGIyLTQ1MjktYjRkNi0zZDRlNjdmMGE5MzMiLCJhdWQiOiJodHRwczovL2F1dGhlbnRpYy1zb3VyY2UuZXhhbXBsZS5pdCIsImV4cCI6MTczMzA1MjYwMCwibmJmIjoxNzMzMDM2NDUwLCJpYXQiOjE3MzMwMzY0MDAsImp0aSI6ImE0YjVjNmQ3LWU4ZjktYWJjZC1lZjEyLTM0NTY3ODkwMTIzNCIsImRub25jZSI6NjUyODQyNDIxMzY4NSwicHVycG9zZUlkIjoiYjJjM2Q0ZTUtZjZnNy1oOGk5LWowazEtbG1ubzEyMzQ1Njc4IiwidXNlcklEIjoiYThiN2M2ZDUtZTRmMy1nMmgxLWk5ajAta2xtbm9wcXJzdHV2IiwibG9hIjoic3Vic3RhbnRpYWwifQ.y42yfMeW2H9h0b0j0BODUml8yF20stY9q3BwoVU5BB90afBj852Q0QlInncdhjXhUjLS1V76cGBxkutDNvxRNA
```